src/commandHandlers/iam.ts

```diff
@@ -76,23 +121,26 @@ export const command = async (
   }
 };
 
-export const description = 'Assign yourself a server role';
+export const description =
+  'Assign yourself a server role or unassign your server role';
 
 export const triggers = ['iam'];
 
-export const usage = `${triggers[0]} <role name> || ${triggers[0]} <role name>, <role name>, ...`;
+export const usage = `${triggers[0]} <add/remove> <role name> or ${triggers[0]} <add/remove> <role name>, <role name>, ...`;
 
 /**
- * Check if the role to assign is the open source role. If so, then add the open source subscription for the user to
+ * Check if the role to assign or remove  is the open source role. If so, then add or remove the open source subscription for the user to
  * receive recurrent messages to remind them to contribute to open source. If not then do nothing.
  * @param roleToAssign
+ * @param toAdd
  * @param message
  */
 async function tryAddOpenSourceUserSubscription(
-  roleToAssign: string,
+  roleToModify: string,
+  toAdd: boolean,
```

Comment:
in general, I think flag arguments are considered bad practice because it insinuates this function does more than one thing. I understand it's easier to do this in this case, but what do you think of splitting up into two functions?
Also, you updated the docs of the function saying it can assign or remove, but the name of the function stayed the same 😃. 

IMO, I think it's less confusing for the reader to split it up into `addOpenSourceUserSubscription` and `removeOpenSourceUserSubscription`, then the condition `roleToModify === config.ROLE.OPEN_SOURCE.name` also goes outside the function, and determines which function to call. What do you think @Vyvy-vi ?

Reply:
that would make this more neater 👍

Reply:
I'll split it into 2 functions tomorrow morning 👍

Reply:
I've split up the `tryaddOpenSourceUserSubscription` function into two. 
Is there some way to improve this further and make the code cleaner?
(personally, I find lots of if-else statements worrying 😅 )